ext/systemd/pe-bolt-server.service

```diff
@@ -10,6 +10,8 @@ EnvironmentFile=-/etc/default/pe-bolt-server-service
 Environment=GEM_HOME=/opt/puppetlabs/server/apps/bolt-server/lib/ruby/gems/2.4.0
 ExecStart=/opt/puppetlabs/server/apps/bolt-server/bin/bolt-server -C /opt/puppetlabs/server/apps/bolt-server/puma_config.rb
 Restart=always
+#set default privileges to -rw-r-----
```

Comment:
I think this is `-rwxr-x---` ?

Reply:
Ah, that must just be for directories though? So since files aren't executable they don't get the executable bit?

Reply:
Correct.